docs/docs/Configuration/configuration-backend-only.md

```diff
@@ -1,154 +1,120 @@
 sidebar_position: 4
 slug: /configuration-backend-only
 ---
 
+Learn how to serve the [Basic Prompting flow](/starter-projects-basic-prompting) as an endpoint in backend-only mode.
 
+Langflow can run in `--backend-only` mode to expose a Langflow app as an API endpoint, without running the frontend UI.
+This is also known as "headless" mode. Running Langflow without the frontend is useful for automation, testing, and situations where you just need to serve a flow as a workload without creating a new flow in the UI.
 
-:::info
+To start Langflow in backend-only mode, pass the `--backend-only` flag at startup.
 
-This page may contain outdated information. It will be updated as soon as possible.
-
-:::
-
-
-
-
-You can run Langflow in `--backend-only` mode to expose your Langflow app as an API, without running the frontend UI.
-
-
-Start langflow in backend-only mode with `python3 -m langflow run --backend-only`.
+```python
+python3 -m langflow run --backend-only
+```
 
+The terminal prints `Welcome to ⛓ Langflow`, and a blank window opens at `http://127.0.0.1:7860/all`.
 
-The terminal prints `Welcome to ⛓ Langflow`, and a blank window opens at `http://127.0.0.1:7864/all`.
 Langflow will now serve requests to its API without the frontend running.
 
+## Prerequisites
 
-## Prerequisites {#81dfa9407ed648889081b9d08b0e5cfe}
+- [Langflow is installed](/getting-started-installation)
+- [You have an OpenAI API key](https://platform.openai.com/)
+- [You have a Langflow Basic Prompting flow](/starter-projects-basic-prompting)
 
-- [Langflow installed](/getting-started-installation)
-- [OpenAI API key](https://platform.openai.com/)
-- [A Langflow flow created](/starter-projects-basic-prompting)
+## Get your flow's ID
 
-## Download your flow's curl call {#d2cf1b694e4741eca07fd9806516007b}
+This guide assumes you have created a [Basic Prompting flow](/starter-projects-basic-prompting) or have another working flow available.
 
-1. Click API.
-2. Click **curl** &gt; **Copy code** and save the code to your local machine.
+1. In the Langflow UI, click **API**.
+2. Click **curl** &gt; **Copy code** and copy the code to your local machine.
 It will look something like this:
 
 ```text
-curl -X POST \\
-    "<http://127.0.0.1:7864/api/v1/run/ef7e0554-69e5-4e3e-ab29-ee83bcd8d9ef?stream=false>" \\
-    -H 'Content-Type: application/json'\\
+curl -X POST \
+    "http://127.0.0.1:7861/api/v1/run/fff8dcaa-f0f6-4136-9df0-b7cb38de42e0?stream=false" \
+    -H 'Content-Type: application/json'\
     -d '{"input_value": "message",
     "output_type": "chat",
     "input_type": "chat",
     "tweaks": {
-  "Prompt-kvo86": {},
-  "OpenAIModel-MilkD": {},
-  "ChatOutput-ktwdw": {},
-  "ChatInput-xXC4F": {}
+  "ChatInput-8a86T": {},
+  "Prompt-pKfl9": {},
+  "ChatOutput-WcGpD": {},
+  "OpenAIModel-5UyvQ": {}
 }}'
-
 ```
 
+In the URL, note the flow ID of `fff8dcaa-f0f6-4136-9df0-b7cb38de42e0`, which is the unique endpoint for your flow.
```

Comment:
```suggestion
In the URL in the code example above, note the flow ID of `fff8dcaa-f0f6-4136-9df0-b7cb38de42e0`, which is the unique endpoint for the flow. Save your flow ID. You will need it later. 
```

Reply:
You don't actually need to save it, i reworded this so it's clearer